libsolidity/codegen/YulUtilFunctions.h

```diff
@@ -72,6 +72,15 @@ class YulUtilFunctions
 	/// Pads with zeros and might write more than exactly length.
 	std::string copyToMemoryFunction(bool _fromCalldata);
 
+	/// @returns a function name that copies string literal to memory
+	/// and returns pointer to memory array containing it
```

Comment:
```suggestion
	/// and returns a pointer to the memory area containing the string literal.
```
is my suggestion correct?

Reply:
It is correct, will update